packages/babel-plugin-transform-modules-systemjs/src/index.ts

```diff
@@ -427,6 +433,11 @@ export default declare((api, options) => {
                   );
                   addExportName(name, name);
                 } else {
+                  if (declar.isVariableDeclaration()) {
+                    // Convert top-level variable declarations to "var",
+                    // because they must be hoisted
+                    declar.node.kind = "var";
+                  }
```

Comment:
A top level var declaration can be nested within for statement:
```js
for (var x = 1;false;);
export { x }
```
or it could be within a do expression.

Reply:
They already have `var`, there is no need to manually change them.